### ISSUE TITLE
`bin/dlog` should promote user to run `mvn install` if the source is not built

### ISSUE THREAD

**Issue by sijie** [apache/bookkeeper#1377]
**BUG REPORT**

1. Please describe the issue you observed:

- What did you do?

1. download the source bindary
2. run `stream/distributedlog/core/bin/dlog local 7000`

- What did you expect to see?

the local emulator started

- What did you see instead?

```
$ ./stream/distributedlog/core/bin/dlog local
JMX enabled by default
DLOG_HOME => ./stream/distributedlog/core/bin/..
Error: Could not find or load main class org.apache.distributedlog.LocalDLMEmulator
```

**Comment by sijie** [comment]
I believe this has been addressed by 7446d5c5d999c88de3cf2b9a36d793295c751a91

**Comment by sijie** [comment]
Fixed by #1407
